src/kudu/consensus/consensus_peers.h

```diff
@@ -66,7 +66,7 @@ class PeerProxy;
 // state aside from the most recent request and response.
 //
 // Peers are also responsible for sending periodic heartbeats
-// to assert liveness of the leader. The peer constructs a heartbeater
+// to assert liveness to the leader. The peer constructs a heartbeater
```

Comment:
It is actually asserting liveness 'of' the leader, right? Followers are not asserting their own liveness to the leader.